catalyst/contrib/models/cv/encoders/resnet.py

```diff
@@ -88,3 +89,23 @@ def forward(self, image):
         """Extract the image feature vectors."""
         features = self.encoder(image)
         return features
+
+
+def adopt_resnet_for_1channel(model: ResNet) -> ResNet:
```

Comment:
could we rename it to `get_resnet1d`? or something like that?

Reply:
done